dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/data/NeuralNetParamServerData.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.dolphin.dnn.data;
+
+import edu.snu.dolphin.dnn.layers.LayerParameter;
+import edu.snu.dolphin.dnn.util.ValidationStats;
+import org.apache.reef.io.network.util.Pair;
+import org.apache.reef.util.Optional;
+
+public final class NeuralNetParamServerData {
```

Comment:
Do we need to have separate classes `NeuralNetParamServerData` and `NeuralNetParamWorkerData`?
`NeuralNetParamServerData` can be replaced with `NeuralNetParamWorkerData` that has `layerParametersList` with only one element.